packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -0,0 +1,175 @@
+
+import '../../database/local_database.dart';
+
+class ConsentAnalytics extends StatelessWidget {
+  const ConsentAnalytics({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Size size = MediaQuery.of(context).size;
+    final LocalDatabase localDatabase = context.watch<LocalDatabase>();
+    final UserPreferences userPreferences = context.watch<UserPreferences>();
+    return Scaffold(
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          SizedBox(
+            height: size.height * 0.2,
+            width: size.width * 0.28,
+            child: Image.asset(
+              'assets/onboarding/data.png',
+              fit: BoxFit.contain,
+            ),
+          ),
+          Align(
+            alignment: Alignment.center,
+            child: Text(
+              'Send anonymous analytics',
+              maxLines: 1,
+              style: TextStyle(
+                  color: Colors.black,
+                  fontWeight: FontWeight.bold,
+                  fontSize: size.height * 0.025),
+            ),
+          ),
+          SizedBox(
+            height: size.height * 0.034,
+          ),
+          ConstrainedBox(
+            constraints: BoxConstraints(
+              maxWidth: size.width * 0.8,
+            ),
+            child: Text(
+              'Help the Open Food Facts volunteer to improve the app.You decide if you want to send anonymous analytics.',
+              textAlign: TextAlign.center,
+              style:
+                  TextStyle(fontSize: size.height * 0.021, color: Colors.black),
+            ),
+          ),
+          SizedBox(
+            height: size.height * 0.03,
+          ),
+          ConstrainedBox(
+            constraints: BoxConstraints(
+              maxWidth: size.width * 0.8,
+            ),
+            child: Text(
+              'If you change your mind this option can be enabled and disabled at any time from the settings.',
+              textAlign: TextAlign.center,
+              style:
+                  TextStyle(fontSize: size.height * 0.021, color: Colors.black),
+            ),
+          ),
+          SizedBox(
+            height: size.height * 0.03,
+          ),
+          //Authorize Button
+          InkWell(
+            borderRadius: BorderRadius.circular(25.0),
+            onTap: () {
+              _analyticsLogic(true, userPreferences, localDatabase, context);
+            },
+            child: Ink(
+              height: size.height * 0.06,
+              width: size.width * 0.7,
+              decoration: BoxDecoration(
+                  color: Colors.green,
+                  borderRadius: BorderRadius.circular(25.0),
+                  boxShadow: <BoxShadow>[
+                    BoxShadow(
+                        blurRadius: 3.0,
+                        color: const Color.fromARGB(144, 0, 0, 0),
```

Comment:
This color is used at least two times. Could you create a constant?

Reply:
This has been solved.